crypto/src/conversation/decrypt.rs

```diff
@@ -26,7 +27,7 @@ pub struct MlsConversationDecryptMessage {
     /// If decrypted message is a commit, this will contain either:
     /// * local pending proposal by value
     /// * proposals by value in pending commit
-    pub proposals: Vec<MlsMessageOut>,
+    pub proposals: Vec<MlsProposalBundle>,
```

Comment:
This change needs to be broadcasted to `crypto-ffi/src/generic.rs|wasm.rs` and to the `CoreCrypto.ts` bindings